grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_form.ts

```diff
@@ -24,9 +26,25 @@ export class OsqueryForm extends FlowArgumentForm<OsqueryArgs> implements
     query: new FormControl(this.defaultQueryDisplayed, Validators.required),
     timeoutMillis: new FormControl(null, Validators.required),
     ignoreStderrErrors: new FormControl(null),
+    fileCollectColumns: new FormControl([]),
   });
 
-  settingsShown = false;
+  fileCollectionSettingsShown = false;
+  lowLevelSettingsShown = false;
+
+  readonly separatorKeysCodes: number[] = [ENTER, COMMA];
+
+  get fileCollectColumnsControl() {
+    const control = this.form.get('fileCollectColumns');
+    if (!control) {
+      throw Error('fileCollectColumns form control is null');
+    }
+
+    return control;
+  }
+  get fileCollectColumns() {
```

Comment:
Same thing about the return type. Having it specified makes the code much easier to read.